demo/reseller/README.md

```diff
@@ -10,4 +10,10 @@ This will copy the required executables and libraries into <fido-iot-src>/demo/r
 
 # Starting the Reseller service
 
-Refer the [Docker Commands](../README.md/#docker-commands) to start the service.
\ No newline at end of file
+Refer the [Docker Commands](../README.md/#docker-commands) to start the service.
+
+***NOTE*** The database file located at <fido-iot-src>/demo/reseller/target/data/reseller.mv.db is not deleted during 'mvn clean'. As as result the database schema and tables are persisted across docker invocations. Please delete the file manually, in case you encounter any error due to the persisted stale data.
```

Comment:
`<fido-iot-src>` not getting rendered into the last line. Prefix the angular brackets with "\" to resolve the issue.